release.py

```diff
@@ -313,6 +316,9 @@ def commit_type_to_category(commit_type: str) -> str:
         'docs': 'Documentation',
         'test': 'Tests',
         'ci': 'CI',
+        'perf': 'Performance',
+        'refact': 'Refactor',
+        'revert': 'Revert',
```

Comment:
I wasn't totally sure how to name these. Maybe "Refactoring" is better than "Refactor"? And "Reversions" rather than "Revert"?